README.md

```diff
@@ -11,12 +9,77 @@ Scripts and cron jobs that can be used to gather helpful information about Shotg
 
 ## Troubleshooting
 
-### Shotgun Log Analysis
-Ruby script that produces a report underlying hot points, heaviest users and scripts, and slowest queries. The output
-of this script is a good starting point to troubleshoot performance issues related to usage patterns. Instructions
+### Shotgun Log Analyzer
+Ruby script that produces a report on underlying psql hot points, heaviest users
+and scripts, and slowest queries. The output of this script is a good starting
+point to troubleshoot performance issues related to usage patterns. Instructions
 in the script header.
 
-[Shotgun Log Analysis](./troubleshooting/shotgun_log_analyzer.rb)
+[Shotgun Log Analyzer](./troubleshooting/shotgun_log_analyzer.rb)
+
+### Log Chop
+
+This ruby script is also a prod log analyzer, but instead of focusing solely on psql queries and query time, it's focused primarily on discovering 'when' and 'what' for investigating the cause of 503 errors, by reporting on the following:
+
+- per-minute summaries of CPU time taken, broken down into Controller, Passenger, CRUD, and psql time. This also includes number of Controller requests per minute.
+- per-user summaries, broken down as above. Report will show top 5 user and top 5 script users for the whole log. the full user summary (not just the top 5) can be generated with the `-c` option.
+- per-user/per-minute summaries. This is a combination of per-minute overall usage PLUS a per-user breakdown per minute in the log file.
+
+Usage help can be obtained with `./log_chop.rb -h`, but here's a quick quide for order of operations:
+
+Parsing a log can be time-consuming if looking at an entire day's worth of activity, so this script is intended to both crop a log file to interesting time periods, and to analyze both full and cropped log files, especially if you already know the timestamps of the 503 incidents. First thing to do is to get the range of the log file in question with the `-r` option.  This option will parse a log and quickly give you the range of time (start and end, in UTC) covered in the log. For example:
+
+```
+./log_chop -r com_shotgunstudio_mystudio_production.log-20190130.gz
+            log begins: 2019-Jan-29 03:50:06 UTC / 2019-Jan-28 19:50:06 PST (-0800)
```

Comment:
`-b` BEGIN_TIME option would be more consistent with this output.

Reply:
changed to `log starts` instead of `log begins`, rather than changing the other part and having to make sure i've changed all the places where the option is referenced :D